baseframe/templates/baseframe.html

```diff
@@ -109,7 +109,7 @@
   {% assets "js_all" -%}
     <script type="text/javascript" src="{{ ASSET_URL }}"></script>
   {%- endassets -%}
-
+  <script type="text/javascript" src='http://maps.google.com/maps/api/js?sensor=false&libraries=places'></script>
```

Comment:
`&` -> `&amp;`

Also, this shouldn't be included unless there's a MapField in the form. Look for this in forms.html.